pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -195,18 +194,14 @@ func (c *ReverseExpandQuery) Execute(
 	req *ReverseExpandRequest,
 	resultChan chan<- *ReverseExpandResult,
 	resolutionMetadata *ResolutionMetadata,
-) {
+) error {
```

Comment:
Can you update the godoc?

```
// Execute yields all the objects of the provided objectType that the given user has, possibly, a specific relation with
// and sends those objects to resultChan. It MUST guarantee no duplicate objects sent.
// This function respects context timeouts and cancellations.
// If an error is encountered (e.g. context timeout) before resolving all objects: the provided channel will NOT be closed, 
// and it will send the error through the channel.
// If no errors, Execute will yield all of the objects on the provided channel and then close the channel
// to signal that it is done.
```